src/main/java/com/github/jcustenborder/kafka/connect/spooldir/AbstractCleanUpPolicy.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright © 2016 Jeremy Custenborder (jcustenborder@gmail.com)
```

Comment:
Just glancing through this and noticed the new files have a copyright year of 2016. Might want to change that :)